server/models/customer.js

```diff
@@ -0,0 +1,31 @@
+//customer: jobs: name, description, required skills, 
+//need date, city, job by id, proposals by id
+
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+const Posts = require('./Posts');
+
+const customerSchema = new Schema({
```

Comment:
Saw you have review under Customer and Contractor, this can be move to User so it's only defined once.